### PR TITLE
Better example for custom Icons for AppBar

### PR DESCRIPTION
**Pull request by brean**
Overview
---
Some icons for the AppBar instead of the generic ObjectCollectionScatter, to look a bit more like the version from the Holograms app

Changes
---
- Fixes: #2310 .

### CODE REVIEW

File: Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
Comment (StephenHodgson) [inline Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg]: why svg instead of png?

Comment (StephenHodgson) [reply Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg]: Oh I see you included both.

Comment (brean) [reply Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg]: Yeah, I added the original files, makes it easier for others to either modify existing icons or add new icons based on these.

Comment (StephenHodgson) [reply Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg]: I think we'll def use TextMeshPro to make the default icon pack for vNEXT.

**Comment by davidkline-ms** [comment]
@cre8ivepark should review the look of these new icons

**Comment by cre8ivepark** [comment]
Thanks! @brean We were not able to include icon glyph font files and images of the glyphs since we cannot publish them in the open-source projects. (font license/design patent issue). 
We provide HoloLens MDL font from Dev Center with proper EULA file. Developers can manually import into Unity and generate the texture from this font file. 
https://docs.microsoft.com/en-us/windows/uwp/design/downloads/

I agree that we need better documentation for this & tools. (something that can automatically generate icon texture from the font file - we had this feature in MixedRealityDesignLabs. Needs porting work)

I'll clone your repo and check out your icon design today. Thank you!
Yoon

**Comment by StephenHodgson** [comment]
@cre8ivepark if we were to generate the icons with a tool like TextMeshPro without including the source font itself, would that satisfy the licensing constraints?

The end result would just be a texture.

**Comment by cre8ivepark** [comment]
@brean I checked out the design. Since your design is very close to the original design, I think it would be better to just use the original design from HoloLens MDL font. I'll apply the updated texture to your repository this weekend. Thanks!

**Comment by cre8ivepark** [comment]
@brean , I applied changes to your repository:
- Updated textures to the original design.
- Changed the location of the textures to existing HoloToolkit/UX/Textures/Icons. Textures under HoloToolkit-Examples folder will break the app bar when the user doesn't have HoloToolkit-Examples folder.
- Removed SVG files. People can use existing icon texture (512x512) png file and [HoloLens MDL font](https://docs.microsoft.com/en-us/windows/uwp/design/downloads/) as a template.
- Updated icon font material to use emission color. (icon was shaded depending on the direction, by the lighting condition)

2018-06-25 06_54_07-unity 2017 2 1p2 64bit - boundingboxgizmoexample unity - mrtk-breen - universa
2018-06-25 06_54_24-unity 2017 2 1p2 64bit - boundingboxgizmoexample unity - mrtk-breen - universa
2018-06-25 06_54_34-unity 2017 2 1p2 64bit - boundingboxgizmoexample unity - mrtk-breen - universa

**Pull request merged**